lib/pluginmanager/pack.rb

```diff
@@ -0,0 +1,60 @@
+require "bootstrap/util/compress"
+require "fileutils"
+
+class LogStash::PluginManager::Pack < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => !LogStash::Environment.windows?
+  option "--zip", :flag, "compress package as a zip file", :default => LogStash::Environment.windows?
+  option "--[no-]clean", :flag, "clean up the generated dump of plugins", :default => true
+  option "--force", :flag, "Overwrite a previously generated package file", :default => false
```

Comment:
I would not do that as they mostly only share options, and this could happen in many other commands. What would be the main concept for you? I see both commands as doing two different things, so I don't see a common stuff except from compression and only for 3 options, I can not see the benefits for now.